### ISSUE TITLE
dev env: Add globalModulePaths in .postcss

### ISSUE THREAD

**Issue by OleksiyRudenko** [kottans/framework-2021#34]
1) In `.postcssrc`: add `globalModulePaths`
```
{
  ...
  plugins: {
    ...
    'postcss-modules': {
      ...
      globalModulePaths: [
        'src/index.css'
      ]
    }
  }
}
```
2) Add `src/index.css` for global styles

**Comment by al0tak** [comment]
It does work, but I added the plugin into `.postcssrc` file, not `.postcss`, since I don't have one.

**Comment by OleksiyRudenko** [comment]
Thank you for confirming and comment on config file. Fixed that

**Comment by OleksiyRudenko** [comment]
Resolved by 74ba98c
